clio-client/src/main/scala/org/broadinstitute/clio/client/dispatch/DeleteExecutor.scala

```diff
@@ -103,10 +103,14 @@ class DeleteExecutor[TI <: TransferIndex](deleteCommand: DeleteCommand[TI])
   )(implicit ec: ExecutionContext): Future[UpsertId] = {
     val pathsToDelete = existingMetadata.pathsToDelete.filter { path =>
       val pathExists = ioUtil.googleObjectExists(path)
-      if (!pathExists) {
-        logger.warn(
-          s"'$path' associated with $prettyKey does not exist in the cloud."
-        )
+      val err = s"'$path' associated with $prettyKey does not exist in the cloud."
```

Comment:
(As seen in the nightly tests) `ioUtil.googleObjectExists` could return false if `gsutil stat` fails for other reasons, so this message won't always be accurate... I'm not sure if we can/should reliably distinguish `gsutil` return codes to fix `googleObjectExists` or if we should just make this message more generic.

Reply:
Does gsutil return something other than just 1? The docs seem to indicate that it is 1 or 0

Reply:
Probably just 1... I'll try to force an auth error and see if it spits out something else.